### ISSUE TITLE
countSubscriptions always returns 1

### ISSUE THREAD

**Issue by alberto-648702** [mroderick/PubSubJS#190]
I was looking at the **countSubscriptions** code and it seems like it always returns 1.

If I'm not misunderstanding the code, the function should be something like this:

```/**
       Count subscriptions by the topic
     * @function
     * @public
     * @alias countSubscriptions
     * @return { number }
    */
    PubSub.countSubscriptions = function countSubscriptions(topic){
        var m;
        var token
        var count = 0;
        for (m in messages) {
            if (Object.prototype.hasOwnProperty.call(messages, m) && m.indexOf(topic) === 0) {
               for (token in messages[m]) {
                 count++;
               }
               break;
            }
        }
        return count;
    };
```

So you have to count the tokens inside the topic object, I'm right?

**Comment by mroderick** [comment]
Thank you for reporting this bug!

I've created #191 to fix the issue with your suggested code.

**Comment by alberto-648702** [comment]
Thank you for the library!

I think that something similar happens to **getSubscriptions** method

**Comment by mroderick** [comment]
This has been fixed with #191 and published to the npm registry as `pubsub-js@1.9.3`

**Comment by mroderick** [comment]
> I think that something similar happens to **getSubscriptions** method

@alberto-648702 would you mind creating a new issue describing what you think is wrong with the `getSubscriptions` method? Bonus points for also creating a pull request that fixes the issue 🏆
